www/src/templates/template-blog-post.js

```diff
@@ -213,7 +213,9 @@ class BlogPostTemplate extends React.Component {
                       (originally published at
                       {` `}
                       <a href={post.frontmatter.canonicalLink}>
-                        {post.frontmatter.publishedAt}
+                        {!post.frontmatter.publishedAt
+                          ? new URL(post.frontmatter.canonicalLink).hostname
```

Comment:
This is not supported by, IE11 without polyfill. I like the suggestion from @jgierer12 about doing the processing in `gatsby-node`, as then we will workaround this annoyance by using that in Node env and not in browser ;)

Reply:
@pieh Yup and `gatsby-node.js` imports the `url` module already. Alright, I'll start working on it.